src/explorer.api/Exploration/Setup/ContextBuilder.cs

```diff
@@ -69,7 +69,8 @@ internal CheckedContext(AircloakConnection connection, string dataSource, string
 
             public DColumnInfo ColumnInfo { get; }
 
-            public Task<DResult<TRow>> Exec<TRow>(DQuery<TRow> query) =>
+            public Task<DResult<TRow>> Exec<TQuery, TRow>(TQuery query)
+            where TQuery : DQueryBuilder, DResultParser<TRow> =>
```

Comment:
This is causing a build error.
```suggestion
            where TQuery : DQuery, DResultParser<TRow> =>
```